README.md

```diff
@@ -51,8 +51,7 @@ Follow these directions to enable query linting during local development so your
     "REACT_APP_GQL_URL": "http://localhost:9090/graphql/query",
     "REACT_APP_API_URL": "http://localhost:3000/api",
     "REACT_APP_UI_URL": "http://localhost:9090",
-    "REACT_APP_SPRUCE_URL": "http://localhost:3000",
-    "REACT_APP_GQL_COOKIE": "cookie-from-REACT_APP_GQL_URL"
+    "REACT_APP_SPRUCE_URL": "http://localhost:3000"
```

Comment:
i tried so many different things to get this to go away. After this PR I am going to delete my fork and re-fork from evergreen/spruce

Reply:
we should reset the changes for this file